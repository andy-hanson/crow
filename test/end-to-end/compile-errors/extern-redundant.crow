main void() trusted
	info log if extern linux
		if extern linux
			"linux linux"
		else
			"just linux"
	else
		"not linux"
