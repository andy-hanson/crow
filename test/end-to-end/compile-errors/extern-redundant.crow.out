exit code: 0
stdout:
info: linux linux
stderr:
test/end-to-end/compile-errors/extern-redundant.crow 3:19-3:24 Extern 'linux' is already in scope, so this expression is always 'true'.
